types/metro-resolver/index.d.ts

```diff
@@ -1,11 +1,14 @@
-// Type definitions for metro-resolver 0.66
+// Type definitions for metro-resolver 0.72
 // Project: https://github.com/facebook/metro
 // Definitions by: Adam Foxman <https://github.com/afoxman>
 //                 Tommy Nguyen <https://github.com/tido64>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
 
 export * from './types';
+export * from './errors';
 
-import { ResolutionContext, Resolution } from './types';
+import { ResolutionContext, Resolution, FileCandidates } from './types';
```

Comment:
This should also be `import type`. We must have missed this when we added the file.

```suggestion
import type { FileCandidates, ResolutionContext, Resolution } from './types';
```